### DISCUSSION TITLE
when I Enable eventing module and save, I got a error

### DISCUSSION THREAD

**Discussion by xtrn143** [spaceuptech/space-cloud#1508] [Q&A] [answered]
The error content: Error saving eventing config
Error 1142: REFERENCES command denied to user 'hx_gsc'@'172.31.40.199' for table 'event_logs'

the user 'hx_gsc' is mySQL username, ip address '172.31.40.199' is K8S master IP, I confused, actually, how to resolve this issue and let me access the table event_logs?

**Comment by sharadregoti** [comment, marked as answer]
@xtrn143 You cannot access the `event_logs` table directly from the frontend. What you can do is just query the rows from the API explorer tab.
```
query {
    event_logs @your_db_alias_here
}
```

Space cloud uses 2 tables for storing eventing info `event_logs` & `invocation_logs`. While setting the eventing config, SC tries to create these tables where `invocation_logs` has a reference on a column of `event_logs` table.
I think this a permission issue. User `hx_gsc` doesn't have the full permissions

**Comment by sharadregoti** [comment]
@YourTechBud What do you think about this issue?

**Comment by YourTechBud** [comment]
It seems like the db user SC provided to SC doesn't have permissions to create a table in the configured database.

The eventing module needs permission to create two table it needs to persist events.
